src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -29,22 +31,25 @@
  * Contains utility methods for populating {@code EduMate} with sample data.
  */
 public class SampleDataUtil {
-    private static final Path SAMPLE_DATA_FOLDER = Paths.get("src", "main", "java", "seedu.address", "model", "util");
     private static final Logger logger = LogsCenter.getLogger(SampleDataUtil.class);
 
     /**
      * Gets sample data from the text file "sampleData.txt"
      * @param size Size of the sample data to be returned.
      */
     public static ReadOnlyEduMate getSampleEduMate(int size) {
-        EduMate sampleAb = new EduMate();
+        EduMate sampleEm = new EduMate();
         try {
-            getSamplePersons().stream().limit(size).forEach(sampleAb::addPerson);
+            List<Person> samplePersons = getSamplePersons();
+            Collections.shuffle(samplePersons);
+            samplePersons.stream()
+                    .filter(Objects::nonNull)
+                    .limit(size).forEach(sampleEm::addPerson);
```

Comment:
Oops! Looks like my Sample Command implementation has seeped into this feature! LMK if you want me to revert.

Reply:
Remember to branch from master next time!